src/actions/actionExport.tsx

```diff
@@ -229,6 +229,7 @@ export const actionExportWithDarkMode = register({
           updateData(appearance === "dark");
         }}
         title={t("labels.toggleExportColorScheme")}
+        showAppearanceBtn
```

Comment:
I left the toggle on here as a way to  swap it before you export, I think that's a reasonable feature but am open to suggestions.